tests/vhost_user_blk_test_server.c

```diff
@@ -1035,8 +1035,9 @@ static void monitor_serve_unix_socket(const char *path,
 
 static void interactive_sigint(int sig)
 {
-    const char *msg = "\nUse 'stop' command or Ctrl+D to exit\n";
-    write(2, msg, strlen(msg));
+    const char *const msg = "\nUse 'stop' command or Ctrl+D to exit\n";
+    ssize_t res = write(2, msg, strlen(msg));
+    VHD_ASSERT(res == strlen(msg));
```

Comment:
Is this safe to use in a signal handler?

Reply:
https://man7.org/linux/man-pages/man7/signal-safety.7.html
Yes. And this only works in the debug build.